Mage.Server/src/test/resources/default_config.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<config>
+    <server serverAddress="address"
+            serverName="name"
+            port="1"
+            secondaryBindPort="2"
+            backlogSize="3"
+            numAcceptThreads="4"
+            maxPoolSize="5"
+            leasePeriod="6"
+            socketWriteTimeout="7"
+            maxGameThreads="8"
+            maxSecondsIdle="9"
+            minUserNameLength="10"
+            maxUserNameLength="11"
+            invalidUserNamePattern="pattern"
+            minPasswordLength="12"
+            maxPasswordLength="13"
+            maxAiOpponents="14"
+            saveGameActivated="true"
+            authenticationActivated="true"
+            googleAccount="account"
+            mailgunApiKey="apikey"
+            mailgunDomain="domain"
+            mailSmtpHost="smtphost"
+            mailSmtpPort="smtpport"
+            mailUser="user"
+            mailPassword="password"
+            mailFromAddress="fromaddress"
+    />
+    <playerTypes>
```

Comment:
You must add settings example and little description to configs (just few words about multihomes param and the link to full description):
* `\Mage.Server\config\config.xml` -- developer config to run;
* `\Mage.Server\release\config\config.xml` -- released config for app distribution.